File: Python/RAM.py
class RAM:

    # RAM attributes.
    __capacity = 0
    __price = 0

    # Constructor.
    # Please take note that after a few try, Python seems to not accept
    # multiple constructor. So I decided to set it as data constructor. #

    # Constructor with RAM's data.
    def __init__(self, capacity, price):
        self.__capacity = capacity
        self.__price = price
    
    # Setter and Getter. #

    # Set capacity attribute.
    def setCapacity(self, capacity):
        self.__capacity = capacity
    
    # Set price attribute.
    def setPrice(self, price):
        self.__price = price
    
    # Get capacity attribute.
    def getCapacity(self):
        return self.__capacity
    
    # Get price attribute.
    def getPrice(self):
        return self.__price

File: Java/RAM.java
public class RAM
{
    // RAM attributes.
    private int capacity;
    private int price;
    
    /* Constructor. */

    // Empty constructor.
    public RAM()
    {

    }

    // Constructor with RAM's data.
    public RAM(int capacity, int price)
    {
        this.capacity = capacity;
        this.price = price;
    }

    /* Setter and Getter. */

    // Set capacity attribute.
    public void setCapacity(int capacity)
    {
        this.capacity = capacity;
    }

    // Set price attribute.
    public void setPrice(int price)
    {
        this.price = price;
    }

    // Get capacity attribute.
    public int getCapacity()
    {
        return this.capacity;
    }

    // Get price attribute.
    public int getPrice()
    {
        return this.price;
    }
}
